variables/Four Column Regex.tpl

```diff
@@ -0,0 +1,342 @@
+        "simpleValueType": true,
+        "name": "convertNullTo",
+        "type": "TEXT"
+      },
+      {
+        "simpleValueType": true,
+        "name": "convertUndefined",
+        "checkboxText": "Convert undefined to…",
+        "type": "CHECKBOX"
+      },
+      {
+        "enablingConditions": [
+          {
+            "paramName": "convertUndefined",
+            "type": "EQUALS",
+            "paramValue": true
+          }
+        ],
+        "displayName": "",
+        "simpleValueType": true,
+        "name": "convertUndefinedTo",
+        "type": "TEXT"
+      },
+      {
+        "simpleValueType": true,
+        "name": "convertTrue",
+        "checkboxText": "Convert true to…",
+        "type": "CHECKBOX"
+      },
+      {
+        "enablingConditions": [
+          {
+            "paramName": "convertTrue",
+            "type": "EQUALS",
+            "paramValue": true
+          }
+        ],
+        "displayName": "",
+        "simpleValueType": true,
+        "name": "convertTrueTo",
+        "type": "TEXT"
+      },
+      {
+        "simpleValueType": true,
+        "name": "convertFalse",
+        "checkboxText": "Convert false to…",
+        "type": "CHECKBOX"
+      },
+      {
+        "enablingConditions": [
+          {
+            "paramName": "convertFalse",
+            "type": "EQUALS",
+            "paramValue": true
+          }
+        ],
+        "displayName": "",
+        "simpleValueType": true,
+        "name": "convertFalseTo",
+        "type": "TEXT"
+      }
+    ]
+  }
+]
+
+
+___SANDBOXED_JS_FOR_WEB_TEMPLATE___
+
+const makeString = require('makeString');
+const inputVar = makeString(data.inputVar);
+const table = data.regexTable;
+const defaultValue = data.defaultValue;
+//Advanced Settings Values
+const ignoreCase = data.ignoreCase;
+const fullMatch = data.fullMatch;
+const capRep = data.captureReplace;
+//Format Value Values
+const convertCaseTo = data.convertCaseTo;
+const convertCase = data.convertCase;
+const convertNullTo = data.convertNullTo;
+const convertNull = data.convertNull;
+const convertUndefinedTo = data.convertUndefinedTo;
+const convertUndefined = data.convertUndefined;
+const convertTrueTo = data.convertTrueTo;
+const convertTrue = data.convertTrue;
+const convertFalseTo = data.convertFalseTo;
+const convertFalse = data.convertFalse;
+//Declare Variables
+  //If you need to change your default delimter, do it here.
+var val = defaultValue !== undefined ? defaultValue.split('|') : defaultValue,
+    a,
+    b;
+if(typeof(table) !=='undefined'){
+  //Itterate through the table
+  for(a=0;a<table.length;a++){
+    if(ignoreCase===true){
+      //Convert regex and input to lowercase
+      var input = inputVar.toLowerCase(),
+          re = table[a].inputVal.toLowerCase();
+    } else{
+      var input = inputVar,
+          re = table[a].inputVal;
+    }
+    if(input.match(re) !== null){
+      if (fullMatch===true){
+        if(input.match(re)[0]==input){
```

Comment:
You're running `input.match()` again, when it's already been run once - that's a redundancy. Better to:

```
const match = input.match(re);
if (match) {
  if (fullMatch) {
    if (match[0] === input) {
...
```